README.md

```diff
@@ -23,350 +22,80 @@
 <img src="assets/github-hr.png" alt="---" />
 <br/>
 
-**k6** is a modern load testing tool, building on [our](https://k6.io/about) years of experience in the load and performance testing industry. It provides a clean, approachable scripting API, [local](https://k6.io/docs/getting-started/running-k6) and [cloud execution](https://k6.io/docs/cloud), and flexible configuration.
+**k6** is a modern load-testing tool, built on [our years of experience](https://k6.io/about) in the performance and testing industries.
+k6 is built to be powerful, extensible, and full-featured. The key design goals is to provide **the best developer experience**.
 
-This is how load testing should look in the 21st century.
+k6 comes with the batteries included:
```

Comment:
```suggestion
Its core features are:
```

Reply:
I'm going to keep this, because "batteries included" is as of now a central design consideration of the application. I actually originally took it out, because it's jargony, but in a separate conversation with @sniku, I realized it's an important part of what k6 tries to be, so I put it back. Still I'd like a less metaphorical description, but let's let it stand for now :-).

Reply:
I do think it's jargony, as you say, but OK. In that case, I think we can drop "the", so "k6 comes with batteries included:".

Reply:
Ah, on second thought I got "writer's remorse" for using a phrase that's metaphorical. Batteries included makes it sound like it could be a CD player, and k6 is not a CD player. I think "full-featured", in the previous paragraph, expresses what we want to say with "batteries included".